podoglyph/gradebook_v
# Development

#### What does this PR do?
Completes initial user_login spec with controllers and namespaced routes for Admin, Teacher, and Student. 

#### How should this be manually tested?
You can create a User with any role and login to confirm the login process takes you to the proper namespaced dashboard.

#### Any background context you want to provide?

#### Questions:

  - Do Migrations Need to be ran?
* No
  - Do Environment Variables need to be set?
* No
  - Any other deploy steps?
* No
